docs/troubleshooting-guide/my-dashboard-is-slow.md

```diff
@@ -0,0 +1,125 @@
+# My dashboard is slow
```

Comment:
Consider adding links to the [Making dashboards faster](https://www.metabase.com/learn/administration/making-dashboards-faster) and [Metabase at scale](https://www.metabase.com/learn/administration/metabase-at-scale.html) in this opening section (in addition to the links below).

Reply:
added